include/nn/hid.h

```diff
@@ -31,13 +31,37 @@ struct ControllerSupportResultInfo {
     int mSelectedId;
 };
 
+struct AnalogStickState {
+    int x, y;
+}
+
+struct NpadBaseState {
+    long mSamplingNumber;
+    long mButtons;
+    AnalogStickState mAnalogStickL;
+    AnalogStickState mAnalogStickR;
+    int mAttributes;
+    int RESERVED;
+};
+
+struct NpadHandheldState : NpadBaseState {};
```

Comment:
```
struct NpadHandheldState {
            int64_t samplingNumber;
            NpadButtonSet buttons;
            AnalogStickState analogStickL;
            AnalogStickState analogStickR;
            NpadAttributesSet attributes;
        };
```
they seem to be redefined rather than relying on inheritance 
also dthe other types they use
```
namespace nn::hid {
struct NpadButton {};
typedef nn::util::BitFlagSet<64, NpadButton> NpadButtonSet;
struct NpadAttribute {};
typedef nn::util::BitFlagSet<32, nn::hid::NpadAttribute> NpadAttributesSet
}
```